website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,166 @@
+:::
+
+## Save Nuke script – the Work File
+Use OpenPype - Work files menu to create a new Nuke script. Openpype offers you the preconfigured naming. 
+![Context](assets/nuke_tut/nuke_AssetLoader.png)
+
+Openpype makes initial setup for your Nuke script. It is the same as running [Apply All Settings](artist_hosts_nuke.md#apply-all-settings) from the OpenPype menu.
+Reads frame range and resolution from Avalon database, sets it in Nuke Project Settings
+Creates Viewer node, sets it’s range and indicates handles by In and Out points
+Reads Color settings from the project configuration, and sets it in Nuke Project Settings and Viewer
+Sets project directory in the Nuke Project Settings to the Nuke Script Directory
+
+:::tip Admin Tip - Workfile Naming
+The [workfile naming](admin_settings_project_anatomy#templates) is configured in anatomy, see **Studio Settings → Project → Anatomy → Templates → Work**
+:::
+
+:::tip Admin Tip - Open Workfile
+You can [configure](project_settings/settings_project_nuke#create-first-workfile) Nuke to automatically open the last version, or create a file on startup. See **Studio Settings → Project → Global → Tools → Workfiles**
+:::
+
+:::tip Admin Tip - Nuke Color Settings
+[Color setting](project_settings/settings_project_nuke) for Nuke can be found in **Studio Settings → Project → Anatomy → Color Management and Output Formats → Nuke**
+:::
+
+## Load plate – Asset Loader
+If your IO or editorial prepared plates and references, or your CG team has a render to be composited, we need to load it.
+
+![OpenPype Load](assets/nuke_tut/nuke_Load.png)
+![Asset Load](assets/nuke_tut/nuke_AssetLoader.png)
+
+Pick the plate asset, right click and choose Load Image Sequence to create a Read node in Nuke. Note that the Read node created by OpenPype is green and has an OpenPype Tab. Green color indicates the highest version available.
+
+## Create Write Node – Instance Creator
+To create OpenPype managed Write node, select the Read node you just created, from OpenPype menu, pick Create.
+In the Instance Creator, pick Create Write Render, and Create.
+
+![OpenPype Create](assets/nuke_tut/nuke_Create.png)
+![OpenPype Create](assets/nuke_tut/nuke_Creator.png)
+
+This will create a Group with a Write node inside.
+
+:::tip Admin Tip - Configuring write node
+You can configure write node parameters in **Studio Settings → Project → Anatomy → Color Management and Output Formats → Nuke → Nodes**
+:::
+
+## What Nuke Publish Does
+From Artist perspective, Nuke publish gathers all the stuff found in the Nuke script with Publish checkbox set to on, exports stuff and raises the Nuke script (workfile) version.
+The Pyblish dialog shows the progress of the process.
+
+![OpenPype Publish](assets/nuke_tut/nuke_Publish.png)
+
+![OpenPype Publish](assets/nuke_tut/nuke_PyblishDialogNuke.png)
+
+#### Publish steps
+- gathers all the stuff found in the Nuke script with Publish checkbox set to on (1)
```

Comment:
The step numbers would look better at the beginning of a sentence.

Reply:
will do